pkgs/development/python-modules/manticore/default.nix

```
@@ -0,0 +1,59 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+, capstone
+, crytic-compile
+, ply
+, prettytable
+, psutil
+, pyelftools
+, pyevmasm
+, pysha3
+, pyyaml
+, rlp
+, setuptools
+, unicorn
+, wasm
+, wrapt
+, z3-solver
+}:
+
+buildPythonPackage rec {
+  pname = "manticore";
+  version = "0.3.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0f4b6yyg6ssmp4hq5njgxvq25iqyjfsps5k3yyjd6nf2y51lxd78";
+  };
+
+  propagatedBuildInputs = [
+    pyyaml
+    wrapt
+    pysha3
+    prettytable
+    pyevmasm
+    rlp
+    ply
+    crytic-compile
+    wasm
+    psutil
+    z3-solver
+    setuptools
+    capstone
+    pyelftools
+    unicorn
+  ];
+
+  patchPhase = ''sed -ie s/pyevmasm==0.2.0/pyevmasm/ setup.py'';
```

```suggestion
  postPatch = ''sed -ie s/pyevmasm==0.2.0/pyevmasm/ setup.py'';
```
For compatibility with (possible) future patches